src/features/MentorPage/MentorCard/MentorCard.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+import { ListCardProps } from '../../../../../mentor_card/src/features/MentorPage/ListCard/types';
```

Comment:
This is the magical type that comes from mentor_card (outside the source folder)